src/examples/details-row-basic/example-details-row-basic.component.html

```diff
@@ -24,9 +24,35 @@
 	            mode="single">
 		<ng-template for="details"
 		             let-$cell>
-			<ul>
-				<li>{{$cell.row.item.color}}</li>
-			</ul>
+			<div class="row-details">
+				<div class="row-details-avatar" [style.background-color]="'#' + $cell.row.item.color">
```

Comment:
remove row-details prefix
rename `row-details` to `box`
reanme row-details-avatar-number to atom-number and other items the same